--- DBD-PgSPI-0.02/PgSPI.h.pg90~	2012-02-15 04:08:46.278041440 +0100
+++ DBD-PgSPI-0.02/PgSPI.h	2012-02-15 04:08:49.001044614 +0100
@@ -14,6 +14,10 @@
 
 /* pull in postgersql headers first */
 #include "postgres.h"
+#include <postgresql/server/utils/bytea.h>
+#include <postgresql/server/catalog/pg_type.h>
+#include <postgresql/server/utils/syscache.h>
+
 #include "funcapi.h"
 #include "executor/spi.h"
 
